src/directives/for.js

```diff
@@ -114,6 +109,9 @@ function addElementInLoopAfterCurrentEl(templateEl, currentEl) {
 }
 
 function lookAheadForMatchingKeyedElementAndMoveItIfFound(nextEl, currentKey) {
+    // We don't know if nextEl is avtually an element so we check that it's not a falsy value first
```

Comment:
// We don't know if nextEl is ~~avtually~~ actually an element so we check that it's not a falsy value first

Reply:
right, I can't spell words correctly, lol. Thanks.